meetings/roles.md

```diff
@@ -14,7 +14,7 @@ The _Scheduler_ sends the calendar invite for the meeting.
 
 **Within 24 Hours of Previous Meeting**
 * [ ] Update the shared calendar series with any new invitees from the last call.  
-* [ ] Visit the [learning path folder].
+* [ ] Visit the [learning path folder]. Make sure this folder is shared with any new attendees from the last call.
```

Comment:
Can we just update the folder so that _anyone_ with the link can access it? Even if we need to make a new folder in order to do it?

![image](https://user-images.githubusercontent.com/9609562/184923543-8ae69f8d-69fe-46fc-b01f-f83cc6510112.png)

Reply:
@tsadler1988 to try out making a new folder.

Reply:
Looks like we didn't get any feedback or success from our approach with @Silona overriding the settings yet?